### PR TITLE
add configuration to better handle regions

### PR DESCRIPTION
**Pull request by infabo**

### CODE REVIEW

**Comment by mmenozzi** [comment]
Hi, thank you for your PR!

File: src/app/code/community/Webgriffe/Alternate/etc/system.xml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                <webgriffe_alternate translate="label" module="webgriffe_alternate">
+                    <label>Alternate [Webgriffe Extension]</label>
+                    <sort_order>501</sort_order>
+                    <show_in_default>1</show_in_default>
+                    <show_in_website>1</show_in_website>
+                    <show_in_store>1</show_in_store>
+                    <fields>
+                        <include_region translate="label comment" module="webgriffe_alternate">
+                            <label>Include Region</label>
+                            <comment>e.g. "en-GB" instead of just "en"</comment>
```
Comment (mmenozzi) [inline src/app/code/community/Webgriffe/Alternate/etc/system.xml]: It's better to have the comment in a CDATA...

File: src/app/code/community/Webgriffe/Alternate/etc/system.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                            <frontend_type>select</frontend_type>
+                            <source_model>adminhtml/system_config_source_yesno</source_model>
+                            <sort_order>10</sort_order>
+                            <show_in_default>1</show_in_default>
+                            <show_in_website>1</show_in_website>
+                            <show_in_store>1</show_in_store>
+                        </include_region>
+                        <override_region translate="label comment" module="webgriffe_alternate">
+                            <label>Override Region</label>
+                            <comment>Region from "general/locale/code" is used by default</comment>
```
Comment (mmenozzi) [inline src/app/code/community/Webgriffe/Alternate/etc/system.xml]: It's better to have the comment in a CDATA...

**Comment by mmenozzi** [comment]
@infabo could you fix my comments, please?

**Comment by infabo** [comment]
You're right. I'll update my PR

**Comment by infabo** [comment]
OK

**Comment by mmenozzi** [comment]
Thank you very much @infabo!

**Pull request merged**